lib/src/dartdoc_options.dart

```diff
@@ -1592,6 +1597,14 @@ List<DartdocOption> createDartdocOptions(
         help: 'The format of documentation to generate: `md` for markdown, '
             '`html` for html.',
         hide: false),
+    DartdocOptionArgOnly<String>('maxFileCount', '0', resourceProvider,
+        help:
+            'The maximum number of files dartdoc is allowed to create (0 for no limit).',
+        hide: false),
+    DartdocOptionArgOnly<String>('maxTotalSize', '0', resourceProvider,
+        help:
+            'The maximum total size dartdoc is allowed to create (0 for no limit).',
+        hide: false),
```

Comment:
I don't think we need these to show up in `--help`, I also don't mind if they do -- I think that's sort of a dartdoc product question @srawlins 
```suggestion
    DartdocOptionArgOnly<String>('maxFileCount', '0', resourceProvider,
        help:
            'The maximum number of files dartdoc is allowed to create (0 for no limit).',
        hide: true),
    DartdocOptionArgOnly<String>('maxTotalSize', '0', resourceProvider,
        help:
            'The maximum total size dartdoc is allowed to create (0 for no limit).',
        hide: true),
```

Reply:
I think they should not appear in `--help`